test/extended/marketplace/marketplace_utils.go

```diff
@@ -0,0 +1,65 @@
+package marketplace
```

Comment:
Isn't this file in your other pr? Should we remove it in one of them and merge one or before the other?


Also these seem like they aren't specific to marketplace. Are functions like these available elsewhere in the origin test package so we can avoid duplication?

Reply:
Yes, not sure which pr will be meged first and have to commit all the needed files for the merged run test .

Reply:
Yes , this file in other pr, but not sure which pr will be merged first , and this file is needed by this pr. I didn't find  so same function available so I think it's ok for marketplace